packages/editor-ui/src/components/RunData.vue

```diff
@@ -845,7 +846,7 @@ export default defineComponent({
 
 			return inputData;
 		},
-		inputData(): INodeExecutionData[] {
+		inputDataAll(): INodeExecutionData[] {
 			let inputData = this.rawInputData;
 
 			if (this.node && this.pinData && !this.isProductionExecutionPreview) {
```

Comment:
Up to you - Untouched but since we're here...

Looks like `this.node` is covered by `this.pinData`?

```suggestion
			if (this.pinData && !this.isProductionExecutionPreview) {
```